scriptparse.py

```diff
@@ -179,7 +179,7 @@ def p_resetsection(p):
 def validate_mcfunction_name(str):
```

Comment:
I know that this isn't code changed it this PR and it definitely should be, but shadowing a builtin is unclean at best.